gnuplot/detailed.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Plot all benchmarks as separate SVGs.
+
+# Make sure gnuplot is installed.
+if ! [ -x "$(command -v gnuplot)" ]; then
+    echo "command not found: gnuplot"
+    exit 1
+fi
+
+# Ensure at least one input and output file is present.
+if [ $# -lt 2 ]; then
+    echo "Usage: gnuplot.sh <INPUT_FILES>... <OUTPUT_DIRECTORY>"
+    exit 1
+fi
+
+# Get last argument as output directory.
+output_index=$#
+output_directory=${!output_index%/}
+mkdir -p "$output_directory"
+
+num_cols=$(cat "$1" | head -n 1 | awk '{ print NF }')
```

Comment:
I guess you can use `wc -w` instead of `awk`?

Reply:
Sure.